.gitlab-ci.yml

```diff
@@ -209,6 +196,7 @@ build-android:
   image:                           parity/rust-android:gitlab-ci
   variables:
     CARGO_TARGET:                  armv7-linux-androideabi
+  dependencies:
```

Comment:
This line prevents artifacts from other jobs to be downloaded before this job.
It makes sense to do the same for the other build jobs.

Reply:
later

Reply:
would suggest to use
```
dependencies: []
```
to make it consistent with the `cache: {}` statement.

Reply:
indeed